src/css.js

```diff
@@ -28,6 +28,7 @@ var
 	// except "table", "table-cell", or "table-caption"
 	// See here for display values: https://developer.mozilla.org/en-US/docs/CSS/display
 	rdisplayswap = /^(none|table(?!-c[ea]).+)/,
+	rnonemptywhite = /^\s+$/,
```

Comment:
Per gh-3072, `\s` is too broad. Rather than introducing a new regex, we should just use `rnotwhite` and let @timmywil's updates thereof perform the final step of this fix.